lib/ansible/modules/system/systemd.py

```diff
@@ -263,6 +263,7 @@
 
 import os
 
+from collections.abs import Iterable
```

Comment:
This would have to be `from ansible.module_utils.common._collections_compat import Iterable`